lib/codecov.js

```diff
@@ -238,11 +316,20 @@ var upload = function(args, on_success, on_failure){
   }
 
   // Detect .bowerrc
-  var bowerrc = execSync('cd '+root+' && test -f .bowerrc && cat .bowerrc || echo ""').toString().trim(), more_patterns = '';
+  var bowerrc;
+  if(!isWindows) {
+    bowerrc = execSync('cd '+root+' && test -f .bowerrc && cat .bowerrc || echo ""').toString().trim(), more_patterns = '';
+  } else {
+    bowerrc = execSync('cd '+root+' && if exist .bowerrc type .bowerrc').toString().trim(), more_patterns = '';
```

Comment:
@eddiemoore 
The variable `more_patterns` is defined at line 11. Are you saying that I should move the definition from line 11 to line 319? Or is it more about removing the repetitive assignment of `more_patterns` with an empty string, by factoring it out to a common place? If so, that's also done on line 11. Please check and advise.